node/resolvers/search/product.ts

```diff
@@ -219,4 +219,15 @@ export const resolvers = {
   OnlyProduct: {
     categoryTree: productCategoriesToCategoryTree,
   },
+  Offer: {
+    spotPrice: (offer: CommertialOffer) => {
+      let spotPrice = offer.Price
+      offer?.Installments.forEach(({NumberOfInstallments, Value}) => {
+        if (NumberOfInstallments === 1 && Value < spotPrice) {
+          spotPrice = Value
+        }
```

Comment:
Do you need to iterate through all the installments or only the first that matches?

Reply:
TBH I am not sure, so I decided to iterate through all of them to be safe, but I can check that!

Reply:
🤷

Reply:
Need to get the lowest value, so it needs to iterate through all